thirdparty/text/README.md

```diff
@@ -0,0 +1,6 @@
+This is an unmodified version of 2 libraries 
+`https://github.com/soasis/idk`
+and
+`https://github.com/soasis/text`
+
+but was removed from their original CMake solution due to issues involving FetchContent
```

Comment:
Can you describe ehat issues those were?

Reply:
No I can't, it wasn't erroring out but just not getting them. It was extremely weird